StRoot/StBFChain/BigFullChain.h

```diff
@@ -1352,7 +1352,7 @@ Bfc_st BFC[] = { // standard chains
   {"PmdUtil"     ,""  ,"","","",                                       "StPmdUtil","Load StPmdUtil",kFALSE},
 
   {"QUtils"      ,""  ,"","PmdUtil,EmcUtil","",                      "","Load QA Libs dependencies",kFALSE},
-  {"Stu"         ,""  ,"","","",                         "StEventUtilities","Load StEventUtilities",kFALSE},
+  {"Stu"         ,""  ,"","","",                 "StTpcDb,StEventUtilities","Load StEventUtilities",kFALSE},
```

Comment:
Why is this change neededr? I found this dependence unnecessary a year ago:
https://www.star.bnl.gov/cgi-bin/protected/viewvc.cgi/cvsroot/StRoot/StBFChain/BigFullChain.h?r1=1.309&r2=1.310

Reply:
So, it looks to me that StEventUtilities does depend on StTpcDb via this call in StRedoTracks.cxx:
https://github.com/star-bnl/star-sw/blob/330b97707cc661275abdac94d90fb43f3e5b420e/StRoot/StEventUtilities/StRedoTracks.cxx#L45-L47

Here are the relevant symbols in the library built with Gcc 11 ⬇️ 
```
.sl79_gcc11/OBJ/StRoot/StEventUtilities/StRedoTracks.o:00000000000007b0 T StRedoTracks::StRedoTracks(char const*, StTpcDbMaker*)
.sl79_gcc11/OBJ/StRoot/StEventUtilities/StRedoTracks.o:00000000000007b0 T StRedoTracks::StRedoTracks(char const*, StTpcDbMaker*)
.sl79_gcc11/OBJ/StRoot/StEventUtilities/StRedoTracks.o:                 U StTpcDbMaker::Class()
.sl79_gcc11/OBJ/StRoot/StEventUtilities/StRedoTracks.o:                 U StMagUtilities::StMagUtilities(StTpcDb*, int)
.sl79_gcc11/OBJ/StRoot/StEventUtilities/StRedoTracks.o:                 U StTpcDb::StTpcDb()
.sl79_gcc11/OBJ/StRoot/StEventUtilities/StRedoTracks.o:0000000000000000 W StTpcDbMaker::tpcDbInterface() const
.sl79_gcc11/OBJ/StRoot/StEventUtilities/StRedoTracks.o:                 U gStTpcDb
```

and with Gcc 4.8.5 ⬇️ 

```
.sl79_gcc485/OBJ/StRoot/StEventUtilities/StRedoTracks.o:0000000000000830 T StRedoTracks::StRedoTracks(char const*, StTpcDbMaker*)
.sl79_gcc485/OBJ/StRoot/StEventUtilities/StRedoTracks.o:0000000000000830 T StRedoTracks::StRedoTracks(char const*, StTpcDbMaker*)
.sl79_gcc485/OBJ/StRoot/StEventUtilities/StRedoTracks.o:                 U StTpcDbMaker::Class()
.sl79_gcc485/OBJ/StRoot/StEventUtilities/StRedoTracks.o:                 U StMagUtilities::StMagUtilities(StTpcDb*, int)
```

Not sure why Gcc 4.8.5 throws away `gStTpcDb`

Do we expect any side effects from loading libStTpcDb.so in BFC as proposed?

Reply:
The output for gcc11 makes sense. The `StTpcDbMaker::tpcDbInterface() const` gets copied over as a weak symbol to `StRedoTracks.o`, so an undefined reference to `gStTpcDb` appears.

Reply:
(I don't mind the BFC change; it appears to be appropriate.)